feat: improve the website with AI 🤖 (#495)
countr/website: content/docs/06-troubleshooting.md

# Troubleshooting

If you're experiencing issues with Countr, don't worry! We're here to help. Check out the common issues below to see if your problem is listed. If it's not, or if you're still having trouble, join our [support server](https://promise.solutions/discord) and we'll assist you further.

## "Remove everyone from a role" action is not working {#role-member-caching}

> If you're having trouble with the "Remove everyone from a role" action, it may be due to a memory-saving feature we use. We don't cache everyone's user information, including their roles. We've tried to work around this issue by fetching everyone in the server that have a score in the counting channel, and then filter out who has the role and who doesn't, but this isn't always reliable. Unfortunately, there's currently no way of fetching everyone from a role or setting a role's members directly through the Discord API.

Reviewer: This could be rewritten to say something about Countr not tracking members, and Discord managing keeping the most accurate count very hard. Very large servers should use the "Give unique role" flow action instead.

Author: This flow action is deprecated anyways so idk why I didn't remove it sooner

```

If Countr deletes messages and reposts like this automatically in your server then check if there's some other bot interfering. One way to check if this is the case is to manually remove the "Manage Messages" permission from Countr in the counting channel, and then try to send a message. If it still deletes then some other bot is interfering, if it doesn't then it's an issue with your setup.

## I don't see the commands. {#permissions}

All commands have a default permission requirement of Administrator, meaning you need to be an Administrator to access commands. This allows the server admins to override and grant whoever they'd like to access to the commands. If you don't see the commands, it's likely because you don't have the necessary permissions. A simple fix is to ask your server admin to grant you access, and to read the [Permissions page](./02-permissions.md) for instructions on how to do so.

If you're a server admin or the server owner, you should have access to all commands by default. If you're still having trouble, or if you have any questions or concerns, please don't hesitate to reach out to us. We're here to help!

## What is Countr's prefix? {#prefix}

> Countr has fully moved over to slash commands, so you can use `/`` as your new prefix! This means that all of Countr's commands are now accessed through the Discord slash command interface. To see a list of available commands, simply type `/`` in any channel where Countr is present.

Reviewer: ```suggestion
Countr uses Discord slash commands, so you can use `/`` as your new prefix! To see a list of available commands, simply type `/`` in any channel where Countr is present.
```
